samcli/lib/utils/resource_trigger.py

```diff
@@ -17,7 +17,7 @@
 from samcli.lib.utils.resources import RESOURCES_WITH_LOCAL_PATHS
 from samcli.local.lambdafn.exceptions import FunctionNotFound, ResourceNotFound
 
-AWS_SAM_FOLDER_REGEX = "^.*\\.aws-sam.*$"
+DEFAULT_WATCH_IGNORED_RESOURCES = ["^.*\\.aws-sam.*$", "^.*node_modules.*$", "^.*package-lock\\.json$"]
```

Comment:
Should we exclude the `package-lock.json` which is in the root folder of `CodeUri`? I thought we should only exclude the one which is inside `node_modules` folder.

Reply:
That's a good call out, I'll double check that this is not actually needed and will remove it.